### PR TITLE
Added italian translation file

### PR DESCRIPTION
**Pull request by CapraTheBest**
Finished the italian translation file.
Please take a look at it, since italian is a bitch to shorten 😢

### CODE REVIEW

File: res/romfs/lang/it.lang [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+
+Enable screen sleep after inactivity
+Attiva riposo schermo per inattività
+
+Remote title key URL(s)
+URL(s) remoto delle title keys
+
+You need to restart freeShop for\n%s to be loaded.\n\nWould you like to do this now?
+Devi riavviare freeShop per\n caricare %s.\n\nDesideri farlo ora?
+Devi riavviare freeShop per\n caricare %s.\n\nDesideri farlo ora?
```
Comment (Cruel) [inline res/romfs/lang/it.lang]: Duplicate line.

Comment (CapraTheBest) [inline res/romfs/lang/it.lang]: fixed

File: res/romfs/lang/it.lang [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+Avvio fallito: 0x%08lX
+
+Failed to resume: 0x%08lX
+Ripresa fallita: 0x%08lX
+
+Failed to suspend: 0x%08lX
+Sopensione fallita: 0x%08lX
+
+Failed to commit title: 0x%08lX
+Falha ao cometer título: 0x%08lX
```
Comment (Cruel) [inline res/romfs/lang/it.lang]: Left over portuguese?

File: res/romfs/lang/it.lang [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+# src/Installer.cpp
+###################
+Failed to install ticket: 0x%08lX
+Installazione ticket fallita: 0x%08lX
+
+Failed to get seed: HTTP %d
+Ricezione seed fallita: HTTP %d
+
+Need FW 9.6+, Seed failed: %016llX
+Firmware 9.6+ necessario, seed fallito: %016llX
```
Comment (Cruel) [inline res/romfs/lang/it.lang]: Would FW still be understood to keep it short?

**Comment by Cruel** [comment]
Yeah, the UI (and screen size) doesn't allow for much space. As long as you got it as short as you can, I will probably end up changing UI a little bit for some languages that are longer.

File: res/romfs/lang/it.lang (hunk 2 above)
Comment (rboninsegna) [inline res/romfs/lang/it.lang]: "Sospensione"

File: res/romfs/lang/it.lang (hunk 3 above)
Comment (rboninsegna) [inline res/romfs/lang/it.lang]: Yep

Comment (CapraTheBest) [inline res/romfs/lang/it.lang]: yeah, but it still fits inside the screen, so I would leave it as it is

File: res/romfs/lang/it.lang (hunk 2 above)
Comment (CapraTheBest) [inline res/romfs/lang/it.lang]: forgot to change it, duh.

**Pull request merged**